src/test/kotlin/com/igorwojda/integer/reverse/challenge.kt

```diff
@@ -4,7 +4,10 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun reverseInt(i: Int): Int {
-    TODO("not implemented")
+    return if (i > 0) {
```

Comment:
Nitpick: you could simply reverse the absolute value as a string and then multiply by the sign of i, right?